entity/c/entity_server.c

```diff
@@ -0,0 +1,51 @@
+#include "c_api.h"
+
+int main() {
+    int serv_sock, clnt_sock;
+    const char *PORT_NUM = "21100";
+
+    struct sockaddr_in serv_addr, clnt_addr;
+    socklen_t clnt_addr_size;
+    serv_sock = socket(PF_INET, SOCK_STREAM, 0);
+    if (serv_sock == -1) {
+        error_handling("socket() error");
+    }
+    memset(&serv_addr, 0, sizeof(serv_addr));
+    serv_addr.sin_family = AF_INET;
+    serv_addr.sin_addr.s_addr = htonl(INADDR_ANY);
+    serv_addr.sin_port = htons(atoi(PORT_NUM));
+
+    if (bind(serv_sock, (struct sockaddr *)&serv_addr, sizeof(serv_addr)) ==
+        -1) {
+        error_handling("bind() error");
+    }
+
+    if (listen(serv_sock, 5) == -1) {
+        error_handling("listen() error");
+    }
+    clnt_addr_size = sizeof(clnt_addr);
+    clnt_sock =
+        accept(serv_sock, (struct sockaddr *)&clnt_addr, &clnt_addr_size);
+    if (clnt_sock == -1) {
+        error_handling("accept() error");
+    }
+    char path[] = "c_server.config";
```

Comment:
This should be passed through the main function's argument.

Reply:
Finished implementing in commit 6f9fd165859b9fc198f30757c6e0341de59fe17a.